worker/scripts/getmake.py

```diff
@@ -0,0 +1,15 @@
+import io, tarfile, urllib.request
+
+def get(url):
+  tar = tarfile.open(fileobj=io.BytesIO(urllib.request.urlopen(url).read()))
+  tar.extractall('worker/out/msys')
+  tar.close()
+
+get('https://sourceforge.net/projects/mingw/files/MSYS/Base/msys-core/msys-1.0.19-1/msysCORE-1.0.19-1-msys-1.0.19-bin.tar.xz/download')
```

Comment:
We need to verify hashes of these downloads, otherwise this is an attack vector or just something that will potentially break our CI in unexplainable way.

Reply:
I've add a hash check.